src/openrct2/ride/CableLift.cpp

```diff
@@ -236,8 +236,8 @@ static bool sub_6DF01A_loop(rct_vehicle* vehicle)
 
         uint16_t trackProgress = vehicle->track_progress + 1;
 
-        const rct_vehicle_info* moveInfo = vehicle_get_move_info(vehicle->var_CD, vehicle->track_type, 0);
-        uint16_t trackTotalProgress = vehicle_get_move_info_size(vehicle->var_CD, vehicle->track_type);
+        const rct_vehicle_info* moveInfo = vehicle_get_move_info(vehicle->TrackSubposition, vehicle->track_type, 0);
```

Comment:
Modify `vehicle_get_move_info` to change the variable names to `trackSubPosition`